content/blog/2019-08-01-new-react-devtools.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Introducing the new React DevTools"
+author: [bvaughn]
+---
+We are excited to announce a new release of the React developer tools, available today in supported browsers!
+
+## What's changed?
+
+A lot has changed!
+[Check out the changelog](https://github.com/bvaughn/react-devtools-experimental/blob/master/CHANGELOG.md) for a complete list.
```

Comment:
People generally expect changelogs to be boring and might not look. But yours is actually pretty beautiful, even with gifs. I'd maybe copy and paste these gifs here, people like to see pretty pictures. Maybe not all of them if you would rather keep it short.

Alternatively you could make a bullet point summary for major changes that make it feel worth it ("a lot has changed" is actually a bit scary), and then conclude "To see the demos and learn about the changes in more detail, see the changelog".

Reply:
I like the last one, bullets with a link.... _and_ a gif?